docs/Module - CI.md

```diff
@@ -1,26 +1,112 @@
-# Module CI
-
-Use the following GitHub workflow to make use of the `module-ci.yaml` reusable workflow. This workflow should be use in every module repository.
-
-Filename: `./github/workflows/module-ci.yaml`
-
-```yaml
-name: module-ci
-on:
-  push:
-    branches:
-      - main
-  pull_request:
-    types:
-      - opened
-      - edited
-      - synchronize
-      - labeled
-      - unlabeled
-jobs:
-  module-ci:
-    uses: cloudeteer/terraform-governance/.github/workflows/module-ci.yaml@main
-    permissions:
-      contents: write
-      pull-requests: read
-```
```

Comment:
the scope of this part is the usage from outside of this repository in our module repositories

Reply:
I agree with @lixhunter; it’s helpful to have a working example here demonstrating how to use the reusable workflow module-ci.yaml. At a minimum, we should include a link to the workflow in the module-templates repository. However, I think including a snippet of the YAML workflow here is valuable as it provides a quick, clear example of how to use our reusable workflow.